ChromiumHtmlToPdfConsole/Options.cs

```diff
@@ -235,9 +235,9 @@ public class Options
     /// <summary>
     ///     Wait for the javascript window.status to equal the given string before starting PDF conversion
     /// </summary>
-    [Option("wait-for-window-status", Required = false, Default = "",
```

Comment:
here and below I removed empty string as default as it doesn't add any value and in will align with other string optional parameters here